tools/azsdk-cli/Azure.Sdk.Tools.Cli/Models/SdkLanguage.cs

```diff
@@ -4,6 +4,8 @@ namespace Azure.Sdk.Tools.Cli.Models;
 
 public enum SdkLanguage
 {
```

Comment:
Can you also add `rust` since rust team is catching up onboarding to this project? CC: @ronniegeraghty